mock-identity-system/README.md

```diff
@@ -18,6 +18,8 @@ Below authentication factors are supported:
 * PWD
 * WLA
 
+For L2 flow, the mock identity system supports only trust-framework as the filter.
```

Comment:
Meaning of L2 flow is unknown to the outer world @Md-Humair-KK . We should clearly callout storage of claim verification metadata. What keys are supported in mock-identity-system. and on which metadata filtering is allowed.